sycl/source/CMakeLists.txt

```diff
@@ -172,19 +179,13 @@ function(add_sycl_rt_library LIB_NAME LIB_OBJ_NAME)
       ${CMAKE_THREAD_LIBS_INIT}
   )
 
-  # Link and include UR
-  target_link_libraries(${LIB_OBJ_NAME}
-    PRIVATE
-      UnifiedRuntime-Headers
-      UnifiedRuntimeCommon
```

Comment:
could you please clarify why UnifiedRuntimeCommon is removed in new version?

Reply:
Two reasons:
- (main) it is not clear why SYCL RT should ever be linked with some "common" UR library - that thing seems to store some shared UR utilities which are used within UR, but not a part of the UR public interface. Tagging @intel/unified-runtime-reviewers to correct me if I'm wrong
- (secondary) object libraries are newer linked with anything, so the only reason to "link" something is to use `INTERFACE`  libraries to get include paths and in #17388 we seems to have an agreement that "common" UR headers should not be in use by SYCL RT

Reply:
yeah I don't think anything in sycl should be using the stuff in UR common, if anything from common *is* used externally the fix will be to make a public interface for it

Reply:
great, thanks for the confirmation!